services/app-web/src/common-code/proto/healthPlanFormDataProto/toDomain.ts

```diff
@@ -186,46 +185,40 @@ function parseProtoDocuments(
     }))
 }
 
-// Just pulling this type out of the hierarchy
-type CapitationRateI =
-    mcreviewproto.ContractInfo.IContractAmendmentInfo['capitationRatesAmendedInfo']
-
-function parseCapitationRates(
-    capRates: CapitationRateI | null | undefined
-): ContractAmendmentInfo['capitationRatesAmendedInfo'] | undefined {
-    if (!capRates) {
-        return undefined
-    }
-
-    return {
-        reason: enumToDomain(
-            mcreviewproto.CapitationRateAmendmentReason,
```

Comment:
I missed this in the AC are we removing capitation rates amended reason entirely as well? I thought this was just about the items being amended list only so a bit confused.

Reply:
Yep. I confirmed this in slack, capitation rates reason is part of the Items Being Amended we're dropping. 

https://cmsgov.slack.com/archives/C014CRU197U/p1652994695062299